string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,21 @@
+/* exported isVowel */
+
+// check if string value matches against any vowel both capital or lower case
+// if yes, return true
+// if no, return false
+
+function isVowel(string) {
+  if (string === 'a' || string === 'A') {
+    return true;
+  } else if (string === 'e' || string === 'E') {
+    return true;
+  } else if (string === 'i' || string === 'I') {
+    return true;
+  } else if (string === 'o' || string === 'O') {
+    return true;
+  } else if (string === 'u' || string === 'U') {
```

Comment:
I would reduce all of this to one `if` block with an expression for each vowel in the condition. Also, you can convert the character to either uppercase or lowercase before the conditional so that you only have to check one set of vowels 🤓